### PR TITLE
svg2png: Clean up code

### PR DESCRIPTION
**Pull request by JSUYA**
Modify the code to fit coding style

### CODE REVIEW

File: src/bin/svg2png/svg2png.cpp [hunk 1]
```diff
@@ -30,30 +30,28 @@ using namespace std;
...
-        if(error) std::cout << "encoder error " << error << ": "<< lodepng_error_text(error) << std::endl;
+        if (error) std::cout << "encoder error " << error << ": "<< lodepng_error_text(error) << std::endl;
     }
 };
 
 
-struct App
-{
+struct App {
     void tvgRender(int w, int h)
```
Comment (hermet) [inline src/bin/svg2png/svg2png.cpp:55]: origin code is correct in thorvg...

File: src/bin/svg2png/svg2png.cpp [hunk 2]
```diff
@@ -160,7 +157,7 @@ struct App
         return 0;
     }
 
-private:
+ private:
     std::string basename(const std::string &str)
```
Comment (hermet) [inline src/bin/svg2png/svg2png.cpp:161]: origin code is correct in thorvg...

**Review by hermet — CHANGES_REQUESTED** [review]

File: src/bin/svg2png/svg2png.cpp (hunk 2 above)
Comment (JSUYA) [reply src/bin/svg2png/svg2png.cpp]: i updated patch

File: src/bin/svg2png/svg2png.cpp (hunk 1 above)
Comment (JSUYA) [reply src/bin/svg2png/svg2png.cpp]: i updated patch

**Review by hermet — APPROVED** [review]

**Pull request merged**